bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -0,0 +1,16 @@
+components:
+  - name: index-management
+    integ-test:
```

Comment:
Just a thought: IntegTests and BwcTests will have the same dependencies for each plugin because they are compile time dependencies and use the same framework. 
Would it be any different for PerformanceTests?

Reply:
yeah, perf test run differently than integ tests and bwc tests which are gradle based.

Reply:
FWIW, we typically wrap that kind of stuff with custom scripts ala build.sh or install.sh.